src/types/util.ts

```diff
@@ -6,4 +6,4 @@ export type CamelizeString<T extends PropertyKey> = T extends string
   : T
   : T;
 
-export type Camelize<T> = { [K in keyof T as CamelizeString<K>]: T[K] }
+export type Camelize<T> = { [K in keyof T]: T[K] };
```

Comment:
This is wrong, if `CamelizeString<K>` is removed, it will not convert the property into camel case.